distage/distage-framework-docker/src/main/scala/izumi/distage/docker/examples/KafkaDocker.scala

```diff
@@ -0,0 +1,34 @@
+package izumi.distage.docker.examples
+
+import distage.{ModuleDef, TagK}
+import izumi.distage.docker.Docker.{ContainerConfig, DockerPort}
+import izumi.distage.docker.{ContainerDef, DockerClientWrapper, DockerContainer}
+import izumi.distage.model.effect.{DIEffect, DIEffectAsync}
+import izumi.logstage.api.IzLogger
+
+object KafkaDocker extends ContainerDef {
+  val primaryPort: DockerPort.TCP = DockerPort.TCP(9092)
+  override def config: Config = {
+    ContainerConfig(
+      image = "wurstmeister/kafka:1.0.0",
```

Comment:
Seems like the latest version is `2.12-2.4.0` , https://github.com/wurstmeister/kafka-docker#tags-and-releases